mysql/mysqldump.go

```diff
@@ -368,10 +402,18 @@ func updateColsByOption(conv *internal.Conv, tableName string, col *ast.ColumnDe
 		case ast.ColumnOptionCheck:
 			column.Ignored.Check = true
 		case ast.ColumnOptionReference:
-			column.Ignored.ForeignKey = true
+			column := col.Name.String()
+			referTable, _ := getTableName(elem.Refer.Table)
```

Comment:
Handle the error? Same for other places.

Reply:
Done

Reply:
I don't see the error handling for getTableName?

Reply:
Apologies, my bad. I missed it. I will update it in next commit.